about.html

```diff
@@ -0,0 +1,227 @@
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#" class="contact">Buy Now</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="about-main">
+      <div class="container">
+        <div class="menu"></div>
+        <div class="nav-below-header">
+          <ul class="nav-below-lists">
+            <li>
+              <a href="index.html"
+                ><i class="fa fa-home" aria-hidden="true"></i
+              ></a>
+            </li>
+            <li>
+              <a href="index.html"><i class="fa fa-angle-right"></i></a>
+            </li>
+            <li><a href="index.html">ግባ</a></li>
+            <li>
+              <a href="index.html"><i class="fa fa-angle-right"></i></a>
+            </li>
+            <li><a href="index.html">እንኳን ደና መጡ</a></li>
+          </ul>
+        </div>
+        <div class="about-main-mobile-title">
+          <q>Wellcome to YayCommerce</q>
+        </div>
+        <section class="about-main-title">
+          <h1>
+            We Are The Largest <br />
+            E-Commerce in Ethiopia
+          </h1>
+        </section>
+        <div class="title"></div>
+        <div class="about-message">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas,
+            totam. Quaerat exercitationem voluptate quis omnis voluptates
+            adipisci dignissimos veniam qui, debitis rerum iure nostrum impedit,
+            corrupti eligendi aliquid possimus vel. Deleniti hic, veritatis,
+            eligendi, modi repellendus beatae rerum quia ipsum illo numquam
+            quibusdam accusamus impedit consequuntur magnam natus. Accusantium,
+            temporibus.
+          </p>
+        </div>
+        <div class="about-contact">
+          <p>
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquid,
+            repellendus explicabo aperiam voluptate nemo a!
+          </p>
+          <i><a href="#">yayner2022@gmail.com</a></i
+          ><i class="fa fa-angle-right"></i>
+        </div>
+      </div>
+    </main>
+    <section class="about-log-section">
+      <div class="container">
+        <h3 class="title">
+          YayCommerce Inc exhibition 2022 Official Logo Competition
+        </h3>
+        <p>
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Qui tempora,
+          necessitatibus tenetur ad modi officiis repudiandae odit quos eos,
+          deserunt mollitia? Distinctio corporis enim praesentium assumenda ab,
+          id fugiat non!
+        </p>
+      </div>
+      <div class="container-log">
+        <div class="about-log">
+          <img
+            src="assets/images/yaycommerceNobg.png"
+            alt="yaycommerce's logo"
+            class="logo-on-about"
+          />
```

Comment:
This image is creating a horizontal scroll bar on the about page. Kindly make it responsive.